client/src/components/advancedSelectWidget/AdvancedSelect.css

```diff
@@ -45,3 +45,11 @@
 .advanced-select-popover > .bp3-overlay > .bp3-transition-container {
   width: 100%;
 }
+
+a.page-link span.sr-only {
+  display: none;
+}
+
+span.page-link span.sr-only {
+  display: none;
+}
```

Comment:
Are these _both_ needed? I only see class `sr-only` being used in `client/src/components/AdvisorReports/Toolbar.js`.